tensorflow/java/src/main/native/graph_jni.cc

```diff
@@ -54,6 +54,30 @@ JNIEXPORT jlong JNICALL Java_org_tensorflow_Graph_operation(JNIEnv* env,
   return reinterpret_cast<jlong>(op);
 }
 
+JNIEXPORT jlongArray JNICALL Java_org_tensorflow_Graph_nextOperation(JNIEnv* env,
+                                                                     jclass clazz,
+                                                                     jlong handle,
+                                                                     jint position) {
+  TF_Graph* g = requireHandle(env, handle);
+  jlongArray rhett = nullptr;
```

Comment:
Might I suggest moving this much lower to where it is needed? So this would be something like:

```c
  TF_Graph* g = requireHandle(env, handle);
  if (g == nullptr) return nullptr;
  ...
  if (operation == nullptr) return nullptr;
  ...
  jlongArray ret = env->NewLongArray(2);
  env->SetLongArrayRegion(ret, 0, 2, handle_and_position);
  return ret;
}
```